CONTRIBUTING.md

```diff
@@ -164,6 +164,25 @@ and add a link to your proposal document (make sure to enable commenting!). For
 weren't able to complete please mention that in the PR description and/or discuss with us on
 [discord](http://discord.unitary.fund).
 
+### List of accepted RFCs
```

Comment:
Should this section be linked in the paragraph above ? The link directs to closed issues on Gihtub. I think the early implementation of the RFC was as an issue when we were still using markdown.

```
Take a look at [previous RFCs](#list-of-accepted-rfcs) for examples on how to fill out your proposal.
```